packages/core/echo/echo-schema/src/query.test.ts

```diff
@@ -179,3 +180,20 @@ describe.skip('Query updates', () => {
     await sleep(10);
   });
 });
+
+test('query with model filters', async () => {
+  const testBuilder = new TestBuilder();
+  const peer = await testBuilder.createPeer();
+
+  const obj = peer.db.add(
+    new Expando({
+      title: 'title',
+      description: new Text('description'),
+    }),
+  );
+
+  expect(peer.db.query().objects).to.have.length(1);
+  expect(peer.db.query().objects[0]).to.eq(obj);
+
+  expect(peer.db.query(undefined, { models: QUERY_ALL_MODELS }).objects).to.have.length(2);
```

Comment:
Let's consider a predicate based DSL for queries, rather than specialized constants.

E.g., 

```ts
db.query(Filter.model(true));
```